### PR TITLE
Script container

### PR DESCRIPTION
**Pull request by TheFynx**
Merging work from https://github.com/ImperialLabs/slapi/issues/23

### CODE REVIEW

File: config/bot.yml [hunk 1]
```diff
@@ -1,2 +0,0 @@
-SLACK_API_TOKEN: 'change me'
-bot_name: 'bot'
```
Comment (aaronblythe) [inline config/bot.yml:2]: Why was this file removed?

File: slapi.rb [hunk 2]
```diff
@@ -10,9 +10,17 @@
...
   # enable :sessions
-
   config_file 'config/environments.yml'
-  config_file 'config/bot.yml'
+
+  # Enable local configs to be ignored
+  # Adds Exception for missing config
+  if File.file?('config/bot.yml')
+    config_file 'config/bot.yml'
+  elsif File.file?('config/bot.local.yml')
```
Comment (aaronblythe) [inline slapi.rb]: Why not both if they exist?  I was thinking the local just comes second so will override the defaults that are checked in.

**Review by aaronblythe — APPROVED** [review]

File: config/bot.yml (hunk 1 above)
Comment (TheFynx) [reply config/bot.yml:2]: It was replaced by this one https://github.com/ImperialLabs/slapi/blob/script_container/config/bot.yml

Comment (aaronblythe) [reply config/bot.yml:2]: Weird the comment should have been from 21 hours ago and the "approved" should have been from 11 minutes ago but it is showing as opposite for me.

I addressed the concern in: https://github.com/ImperialLabs/slapi/pull/31/commits/deae9a8017aad30d9365a280b16c39010ddf6397

I am now curious where my other comment went?
